index.bs

```diff
@@ -1230,8 +1230,16 @@ To <dfn for="SendStream">abort</dfn> a {{SendStream}} |stream| with |reason|, ru
 
 1. Let |transport| be |stream|'s [=[[Transport]]=].
 1. Let |promise| be a new promise.
-1. Let |code| be TBD.
+1. Let |code| be 0.
 1. [=set/Remove=] |stream| from |transport|'s [=[[SendStreams]]=].
+1. If |reason| is a {{WebTransportError}} and |reasons|'s [=[[ApplicationProtocolCode]]=] is not
```

Comment:
ApplicationProtocolCode might be a poor choice of name as that is a term QUIC uses for the 62-bit value.  What about WebTransportProtocolErrorCode?  (It's long, I know.)